scripts/scalability-testing/config_builder.py

```diff
@@ -67,12 +67,14 @@
 
 class DeviceConfig(object):
```

Comment:
We should probably add some documentation to these classes as they are hopefully going to be used by external entities.